src/ast/writing/ast_mapping.ts

```diff
@@ -102,6 +102,16 @@ function descTrimRight(desc: SrcDesc): void {
     }
 }
 
+function isSpdxLicence(desc: SrcDesc): boolean {
```

Comment:
Maybe rename this function to `hasSpdxLicense`? Not a blocker

Reply:
Fixed in 214a471